backend/api/Areas/Admin/Profiles/Parcel/BuildingProfile.cs

```diff
@@ -31,11 +36,13 @@ public BuildingProfile()
 
             CreateMap<Entity.Building, Model.BuildingModel>()
                 .ForMember(dest => dest.BuildingConstructionType, opt => opt.MapFrom(src => src.BuildingConstructionType.Name))
```

Comment:
Will these fail if any of the properties are null?

Reply:
I tested with the minimal allowable set of the new properties.